src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1842,7 +1870,7 @@ void goto_instrument_parse_optionst::help()
     HELP_REMOVE_CALLS_NO_BODY
     HELP_REMOVE_CONST_FUNCTION_POINTERS
     " --add-library                add models of C library functions\n"
-    " --model-argc-argv <n>        model up to <n> command line arguments\n"
+    HELP_ARGC_ARGV
```

Comment:
Don't worry about fixing this for this particular PR but in general we try to make sure that the code compiles after every commit.  That way we can use `git bisect` more easily.